test/checks/aria/aria-prohibited-attr.js

```diff
@@ -9,39 +9,79 @@ describe('aria-prohibited-attr', function() {
     checkContext.reset();
```

Comment:
Need tests for checking when the messageKey is set to `noRoleSingular` when returned as `false` and `undefined`